cyral/provider.go

```diff
@@ -91,6 +92,12 @@ func Provider() *schema.Provider {
 			"cyral_integration_looker":          resourceIntegrationLooker(),
 			"cyral_integration_microsoft_teams": resourceIntegrationMsTeams(),
 			"cyral_integration_pager_duty":      resourceIntegrationPagerDuty(),
+			"cyral_integration_saml_adfs":       resourceIntegrationSAML("adfs-2016"),
+			"cyral_integration_saml_aad":        resourceIntegrationSAML("aad"),
+			"cyral_integration_saml_forgerock":  resourceIntegrationSAML("forgerock"),
+			"cyral_integration_saml_gsuite":     resourceIntegrationSAML("gsuite"),
+			"cyral_integration_saml_okta":       resourceIntegrationSAML("okta"),
+			"cyral_integration_saml_pingone":    resourceIntegrationSAML("pingone"),
```

Comment:
Instead of having multiple resources, why not having just one `cyral_integration_saml` where we set a parameter `idp_provider` with values `adfs-2016`, `aad`, etc? We can define a consistency check to avoid users adding options that are not supported. If you agree and there are no technical impediments, please implement this.